packages/trip-details/src/trip-detail.tsx

```diff
@@ -51,19 +51,14 @@ export default class TripDetail extends Component<Props, State> {
             </S.ExpandButton>
           )}
         </S.TripDetailSummary>
-        <VelocityTransitionGroup
-          enter={{ animation: "slideDown" }}
-          leave={{ animation: "slideUp" }}
-        >
-          {expanded && (
-            <S.TripDetailDescription>
-              <S.HideButton onClick={this.onHideClick}>
-                <TimesCircle size="0.92em" />
-              </S.HideButton>
-              {description}
-            </S.TripDetailDescription>
-          )}
-        </VelocityTransitionGroup>
+        <AnimateHeight duration={300} height={expanded ? "auto" : 0}>
+          <S.TripDetailDescription>
```

Comment:
Previously, the contents in trip details was rendered conditionally. From my local test log, it looks there is still a reference to momentjs that is lurking... could you try remove it?